lib/jekyll-github-metadata/value.rb

```diff
@@ -39,14 +42,6 @@ def render
         raise e
       end
 
-      def to_s
-        render.to_s
-      end
-
-      def to_json(*)
```

Comment:
Because it's `def_delegators` (plural), I believe it will delegate `Value#to_json` to calling `to_json` on the result of `#render`, meaning it should accept the second argument so long as the result of `#render` does.